domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/display/DisplayGroupAdapterService.java

```diff
@@ -59,7 +64,7 @@ public WizardPageCollection findWizardPagesByCaseTypeId(String caseTypeReference
 
                     List<String> allSubTypePossibilities =
```

Comment:
I might be wrong but `allSubTypePossibilities` is initialised on line 65 but then used only on line 102. Can we move it close to the point of usage please

Reply:
It is initialized outside of the forEach loop to avoid unnecessary cycles.